.tmux.conf

```diff
@@ -1,7 +1,63 @@
+# cat << EOF > /dev/null
+# This file is a shell/tmux.conf polyglot
+# https://en.wikipedia.org/wiki/Polyglot_(computing)
+
 set exit-unattached off
 set exit-empty off
 set -g status-position top
 
 set -g default-terminal "screen-256color"
 set -g mouse on
 setw -g xterm-keys on
+set -g status-keys vi
```

Comment:
Could you add an inline comment for those two? I cannot understand the difference between those two without looking into `tmux` documentation.